torch/csrc/jit/script/compiler.cpp

```diff
@@ -2636,14 +2636,24 @@ struct to_ir {
       args.emplace_back(loc, "end", emitExpr(Expr(slice.end().get())));
     }
     if (input->type()->cast<TupleType>()) {
+
+      auto has_step = slice.step().present();
+      // TODO: add support for slicing tuples
+      if (has_step)
+      {
+        throw ErrorReport(loc) << "Unsupported operation: slicing tuples isn't supported";
```

Comment:
I think this error message is not accurate, right? We do support tuple slices, just not steps

Reply:
will fix.